docs/_includes/component.njk

```diff
@@ -1,434 +1,376 @@
-{% extends "default.njk" %}
-{# Find the component based on the `tag` front matter #}
-{% set component =
-getComponent('sl-' + page.fileSlug) %}
-{% block content %}
-  {# Determine the badge variant for code status #}
-  {% if
-  component.status == 'stable' %}
-    {% set badgeVariant = 'primary' %}
-  {% elseif component.status == 'experimental' %}
-    {% set badgeVariant = 'warning' %}
-  {% elseif component.status == 'planned' %}
-    {% set badgeVariant = 'neutral' %}
-  {% elseif
-component.status == 'deprecated' %}
-    {% set badgeVariant = 'danger' %}
-  {% else %}
-    {% set badgeVariant = 'neutral' %}
-  {% endif %}
-  {# Determine the badge variant for pattern status #}
-  {% if component.pattern.length %}
-    {% if component.pattern == 'stable' %}
-      {% set patternBadgeVariant = 'primary' %}
-    {% elseif component.pattern == 'in review' %}
-      {% set
-      patternBadgeVariant = 'neutral' %}
-    {% else %}
-      {% set patternBadgeVariant = 'danger' %}
-    {% endif %}
-  {% else %}
-    {% set
-    patternBadgeVariant = 'danger' %}
-  {% endif %}
-  {# Determine the badge variant for figma status #}
-  {% if
-  component.figma.length %}
-    {% if component.figma == 'ready' %}
-      {% set figmaBadgeVariant = 'success' %}
-    {% elseif
-component.figma == 'draft' %}
-      {% set figmaBadgeVariant = 'warning' %}
-    {% else %}
-      {% set figmaBadgeVariant = 'danger' %}
-    {% endif %}
-  {% else %}
-    {% set figmaBadgeVariant = 'danger' %}
-  {% endif %}
-  {# Header #}
-  <header class="component-header">
-    <div class="component-header__info">
-      <sl-tag variant="neutral" size="small" pill> Since Shoelace {{component.since or '?' }}
-      </sl-tag>
-      <sl-tag size="small" variant="{{ badgeVariant }}" pill style="text-transform: capitalize">
+{% extends "default.njk" %} {# Find the component based on the `tag` front matter #} {% set component =
```

Comment:
This happened because VS code didn't auto-select `nunjucks` as the language mode & it reformatted the entire page automatically as HTML after I made a small change. Realized I can manually set the language mode to `nunjucks` to put the formatting back, so this is now fixed on the new rebased branch I just pushed